src/GitTools.Testing/Fixtures/RepositoryFixtureBase.cs

```diff
@@ -33,7 +33,7 @@ public string RepositoryPath
 
         public SequenceDiagram SequenceDiagram
         {
-            get { return _sequenceDiagram; }
+            get { return this.sequenceDiagram; }
```

Comment:
I forgot to handle properties and indexers. Shall I push new changes or do a follow-up PR?

Reply:
add to this pr